tools/mo/openvino/tools/mo/middle/StridedSliceNormalizer.py

```diff
@@ -113,6 +113,12 @@ def find_and_replace_pattern(self, graph: Graph):
             if node.is_in_port_connected(3):
```

Comment:
Please check that attributes of this operation would not be adjusted to NCHW layout by this code above:
```
            PermuteAttrs.create_permute_attrs(node,
                                              attrs=[('begin_mask', 'input:0'),  # but indeed depends from slice_rank
                                                     ('end_mask', 'input:0'),
                                                     ('new_axis_mask', 'input:0'),
                                                     ('shrink_axis_mask', 'input:0'),
                                                     ('ellipsis_mask', 'input:0')])
```

Reply:
It looks that permuted attrs are saved in additional node attribute but they won't applied since  node['nchw_layout'] = True